constants/environment.js

```diff
@@ -0,0 +1,11 @@
+const DEV = 'dev';
+const DEVELOPMENT = 'development';
+const PROD = 'prod';
+const PRODUCTION = 'production';
+
+export {
+    DEV,
+    DEVELOPMENT,
+    PROD,
+    PRODUCTION
+}
```

Comment:
why don't create one object with all constants and export it? To do not copy paste the constant name from definition to export object?